datadog_lambda/tracing.py

```diff
@@ -43,9 +43,15 @@
 
 logger = logging.getLogger(__name__)
 
+AWS_LAMBDA_FUNCTION_NAME = "AWS_LAMBDA_FUNCTION_NAME"
+DD_SERVICE = "DD_SERVICE"
+
 dd_trace_context = {}
 dd_tracing_enabled = os.environ.get("DD_TRACE_ENABLED", "false").lower() == "true"
 
+function_env_var = os.environ.get(AWS_LAMBDA_FUNCTION_NAME, "function")
+service_env_var = os.environ.get(DD_SERVICE, function_env_var)
+
```

Comment:
Just so I understand peer.service tag is resolution is (in the order of priority)
1. Val of DD_SERVICE
2. Val of AWS_LAMBDA_FUNCTION_NAME
3. "function"

I didn't read much of the peer.service rfc but again i'd like to know more about AWS_LAMBDA_FUNCTION_NAME

Reply:
I just did this due to the profiler in Python, not really the best option, but it is what we have.

Reply:
Hmm something isn't clicking, how does this relate to the profiler?